eris_test.go

```diff
@@ -314,6 +314,24 @@ func TestErrorIs(t *testing.T) {
 			compare: eris.Wrap(globalErr, "additional context"),
 			output:  true,
 		},
+		"wrapped error against a wrapped error with a different order": {
+			cause:   eris.New("root error"),
+			input:   []string{"additional context", "even more context"},
+			compare: eris.Wrap(eris.New("additional context"), "root error"),
+			output:  false,
+		},
+		"wrapped error against a root error": {
+			cause:   eris.New("root error"),
+			input:   []string{"additional context", "even more context"},
+			compare: eris.New("additional context"),
+			output:  true,
+		},
+		"wrapped error against another wrapped error": {
```

Comment:
this is essentially mapping to the case where a: b: c if matched against a: b or b: c will return true?

Reply:
yeah, that's the idea. although i don't think this currently works for `a: b`. do you think it should?

Reply:
why doesn't it work for a: b case? because unwrapping 'c' will return false? I think it should work otherwise there is a false expectation around Is method's functionality. One way to make it work could be to keep unwrapping wrapped error without returning until root error is hit and then compare with unwrapped values. Similar to backtracking algo.

Reply:
that sounds like a valid approach, i'll give it a shot tomorrow